api/v1beta1/namespacedverifier_types.go

```diff
@@ -49,6 +53,14 @@ type NamespacedVerifierSpec struct {
 	Parameters runtime.RawExtension `json:"parameters,omitempty"`
 }
 
+// GetType returns verifier spec type and is backward compatible with the old name field
+func (spec *NamespacedVerifierSpec) GetType() string {
```

Comment:
it would be good to have one simple unit test.

Reply:
Moved this function to `cutils` pkg for testing